release/stable/windowsservercore20H2/meta.json

```diff
@@ -17,5 +17,5 @@
         "size": 1
     },
     "UseAcr": true,
-    "IsBroken": true
+    "IsBroken": false
```

Comment:
```suggestion
```

Reply:
Oh yeah, that actually is a better idea.  How did I not think of that 😄